ui/app/styles/core/footer.scss

```diff
@@ -7,5 +7,9 @@
   span:not(:first-child) {
     display: inline-block;
     padding: 0 0.5rem;
+
+    @include until($mobile) {
```

Comment:
What do you think about stacking things in the footer instead of hiding them? Version / upgrade isn't very important imo, but keeping the docs link would be nice. Either way works though!

Reply:
I thought about that, but more likely would be to collapse them down to icons or something. This now matches what a couple of the other apps are doing, so I think consistent is good until we figure out something official in the design system.